docker-compose.yml

```
@@ -43,12 +43,13 @@ services:
       - mq
 
   mercury-service-api:
-    image: 258143015559.dkr.ecr.us-east-1.amazonaws.com/mercury/api
+    platform: linux/x86_64
```

```suggestion
    platform: linux/amd64
```

Best to use more standard platform tag. Will build the same